src/vcpkg/commands.setinstalled.cpp

```diff
@@ -20,11 +20,19 @@ namespace vcpkg::Commands::SetInstalled
     static constexpr StringLiteral OPTION_ONLY_DOWNLOADS = "only-downloads";
     static constexpr StringLiteral OPTION_WRITE_PACKAGES_CONFIG = "x-write-nuget-packages-config";
     static constexpr StringLiteral OPTION_NO_PRINT_USAGE = "no-print-usage";
+    static constexpr StringLiteral OPTION_USE_HEAD_VERSION = "head";
+    static constexpr StringLiteral OPTION_EDITABLE = "editable";
```

Comment:
I'm somewhat opposed to these given what x-set-installed is intended to do. We can't 'make the world be exactly this list' if we are `--head`.

I'm all for `--enforce-port-checks` and `--allow-unsupported` though.

Reply:
I have already expected that this is controversial. I personally don't see a conflict between "make the world be exactly this list" and head mode. Then it is "make the world be exactly this list with head builds".